client/contributing.md

```diff
@@ -0,0 +1,131 @@
+
+# Contributing
+
+The following is a set of guidelines for contributing to Portfolio-1, which is hosted on [Github](https://github.com/rwiteshbera/Pixby/). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```

Comment:
What do you mean by `Portfolio-1`? Have you copied the docs from another repo. Please write in your own way. and also do not forget to link it with README.md. Thank you.